src/sdk-reference/js/6/kuzzle/introduction/index.md

```diff
@@ -10,6 +10,8 @@ order: 0
 The Kuzzle class is the main class of the SDK.  
 Once instantiated, it represents a connection to your Kuzzle server.
 
+It's inherits from [KuzzleEventEmitter]({{ site_base_path }}sdk-reference/js/6/kuzzle-event-emitter).
```

Comment:
I think it's better to move this at the beginning of the description. At the very least, the wording should be fixed: `It inherits from ....`